src/blog/2019-03-14/announcing-dotnet-framework-support.pug

```diff
@@ -0,0 +1,67 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+    
+        Stryker just stepped up to become the standard for mutation testing .NET. With version 0.10 you can finaly test your tests on .NET Framework projects. This was a highly needed feature since most .NET projects still are running on .NET Framework and not on the new .NET Core.
+
+        ## Upgrade to 0.10
+        If you have been using Stryker.NET before you should notice the installation has changed. Please follow these steps to correctly upgrade and keep your Stryker up to date.
+        
+        #### 1. Installation
+        To install stryker **globally** (.NET CORE 2.2+):
+
+        `dotnet tool install -g dotnet-stryker`
+
+        To install stryker **locally** (.NET Core 3.0+): 
+        Create a file called dotnet-tools.json in your project folder. You can checkin to version control to make sure all team members have access to stryker
+
+        `dotnet new tool-manifest` 
+
+        Then install stryker without the -g flag while executing the following command in the project folder
+
+        `dotnet tool install dotnet-stryker`
+
+        #### 2. Install dependencies
+        For .NET Core projects this step can be skipped.
+        
+        For .NET Framework projects, make sure [nuget.exe](https://docs.microsoft.com/en-us/nuget/install-nuget-client-tools#windows) is installed on your system. Please follow their installation instructions.
+
+        #### 3. Remove old references
+        The old way of installing stryker is deprecated and replaced by the dotnet core tool. You can remove the old references from your test project since they won't be updated anymore.
+
+
+        ~~`<ItemGroup>`~~
+
+        ~~`<DotNetCliToolReference Include="StrykerMutator.DotNetCoreCli" Version="0.9.0" />`~~
+
+        ~~`<PackageReference Include="StrykerMutator.DotNetCoreCli" Version="0.9.0" />`~~
+
+        ~~`</ItemGroup>`~~
+
+        #### 4. Have fun using stryker!
+        You can now kick off stryker using `dotnet stryker` or `dotnet-stryker`
+
+        If stryker is installed as a dotnet core 3.0 local tool, use the following instead:
+
+        `dotnet tool run dotnet-stryker`
+
+        ## The challenges
```

Comment:
The title here is "challenges". However, it is structured as a question and answer segment. When reading them, I can vaguely understand what the challenges were. However, I think it would be clearer if we it down in sentences. That way I don't have to guess.